libsplinter/src/node_registry/mod.rs

```diff
@@ -21,9 +21,9 @@ use std::collections::HashMap;
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
 pub struct Node {
-    /// The Splinter identity of the node.
```

Comment:
As these are public fields, you should keep and update the doc comments, not remove them.

Reply:
Fixed